ui/src/components/Entity/EntityTagsMode.jsx

```diff
@@ -33,6 +33,12 @@ class EntityTagsMode extends React.Component {
   render() {
     const { intl, entity, tags } = this.props;
 
+    if (tags.shouldLoad || tags.isLoading) {
```

Comment:
I think this is the wrong place to implement this, because it's still going to show the top menu in an incomplete way. Can we do the section loading thing in `EntityViews` and make it dependent on references, tags etc.?